autoload/ctrlp/funky/ft/fish.vim

```diff
@@ -0,0 +1,26 @@
+" Language: Fish shell
+" Author: melhakim
+" License: The MIT License
+
+function! ctrlp#funky#ft#fish#filters()
+  " TODO - Add following options:
+  " -d DESCRIPTION or --description=DESCRIPTION
+  " -w WRAPPED_COMMAND or --wraps=WRAPPED_COMMAND
+  " -e or --on-event EVENT_NAME
+  " -v or --on-variable VARIABLE_NAME
+  " -j PID or --on-job-exit PID
+  " -p PID or --on-process-exit PID
+  " -s or --on-signal SIGSPEC
+  " -S or --no-scope-shadowing
+  " -V or --inherit-variable NAME
```

Comment:
I just didn't know these options are accepted in a function line in fish script, LGTM anyway though.
https://fishshell.com/docs/current/cmds/function.html